File: node/models/userLog.js
const { DataTypes } = require('sequelize');
const sequelize = require('../config/database');

const UserLog = sequelize.define('UserLog', {
    id_user: { type: DataTypes.INTEGER, primaryKey: true, references: { model: 'users', key: 'id' } },
    email: { type: DataTypes.STRING(50), allowNull: false },
    created_at: { type: DataTypes.DATE, allowNull: false, defaultValue: DataTypes.NOW },
    logs_status: { type: DataTypes.TEXT }
}, { tableName: 'user_log', timestamps: true });

module.exports = UserLog;

File: app/models/user_log.py
from sqlalchemy import Column, Integer, String, TIMESTAMP, Text, ForeignKey
from app.database.base import Base

class UserLog(Base):
    __tablename__ = "user_log"
    id_user = Column(Integer, ForeignKey("users.id"), primary_key=True)
    email = Column(String(50), nullable=False)
    created_at = Column(TIMESTAMP, nullable=False)
    logs_status = Column(Text)
